### PR TITLE
Refs #25706 - Removed inline JavaScript from OpenLayers template.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: django/contrib/gis/templates/gis/openlayers.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 <div id="{{ id }}_div_map">
     <div id="{{ id }}_map"></div>
-    {% if not disabled %}<span class="clear_features"><a href="javascript:{{ module }}.clearFeatures()">{% translate "Delete all Features" %}</a></span>{% endif %}
+    {% if not disabled %}<span class="clear_features"><a href="">{% translate "Delete all Features" %}</a></span>{% endif %}
```
Comment (felixxm) [inline django/contrib/gis/templates/gis/openlayers.html:12]: Can we do the same for `contrib/gis/templates/gis/admin/openlayers.html`?

File: django/contrib/gis/static/gis/js/OLMapWidget.js [hunk 2]
```diff
@@ -107,6 +107,13 @@ ol.inherits(GeometryTypeControl, ol.control.Control);
         if (initial_value && !this.options.is_collection) {
             this.disableDrawing();
         }
+        const clearNode = document.getElementById(this.map.getTarget()).nextElementSibling;
+        if (clearNode.classList.contains("clear_features")) {
```
Comment (felixxm) [inline django/contrib/gis/static/gis/js/OLMapWidget.js:111]:
```suggestion
        if (clearNode.classList.contains('clear_features')) {
```

**Review by felixxm — COMMENTED** [review]
@claudep Thanks :+1:

File: django/contrib/gis/templates/gis/openlayers.html (hunk 1 above)
Comment (claudep) [reply django/contrib/gis/templates/gis/openlayers.html:12]: I don't plan to touch those admin contents until they die in Django 5.0. People giving attention to CSP can always switch to the new GISModelAdmin.

**Comment by felixxm** [comment]
buildbot, test on selenium.

**Pull request merged**